plugins/source/azure/resources/services/iothub/hubs.go

```diff
@@ -1,891 +0,0 @@
-package iothub
-
-import (
-	"context"
-	"encoding/json"
-
-	"github.com/Azure/azure-sdk-for-go/services/iothub/mgmt/2021-07-02/devices"
-	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
-	"github.com/cloudquery/cq-provider-sdk/provider/diag"
-	"github.com/cloudquery/cq-provider-sdk/provider/schema"
-)
-
-func IothubHubs() *schema.Table {
```

Comment:
Moved to `plugins/source/azure/resources/services/iothub/devices.go` and now called `devices`